### PR TITLE
AST-92 - refactor: Updated Astra Options page with Starter Templates screenshot & Important Links postbox

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Reduced other postboxes on Astra options page & managed with single "Important Links" postbox
- Starter Templates plugin screenshot updation

### Screenshots
- Updated postbox of Starter Template - https://a.cl.ly/8LubBx6G

### Types of changes
- Non-breaking changes

### How has this been tested?
- Check with Astra options page
- Test with other browser can be a test case as well.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by Nikschavan** [comment]
Task linked: [AST-3347 Update Astra Options Page with new starter templates screenshot & optimizing postboxes](https://app.clickup.com/t/3387275/AST-3347)

**Review by rohitp-bsf — APPROVED** [review]

File: inc/core/class-astra-admin-settings.php [hunk 1]
```diff
@@ -586,6 +584,72 @@ public static function general_page() {
 			require_once ASTRA_THEME_DIR . 'inc/core/view-general.php';// phpcs:ignore WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound
 		}
 
+		/**
+		 * External links through Astra Options page.
+		 *
+		 * @since x.x.x
+		 */
+		public static function astra_external_links_scetion() {
```
Comment (vrundakansara) [inline inc/core/class-astra-admin-settings.php:592]: Let's remove the `astra` prefix from the function name. Also, rectify the `external_links_section` typo.

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: inc/core/class-astra-admin-settings.php (hunk 1 above)
Comment (imnavanath) [reply inc/core/class-astra-admin-settings.php]: Done 👍 Also updated func name with - `external_important_links_section`

**Review by vrundakansara — APPROVED** [review]

File: inc/core/class-astra-admin-settings.php [hunk 2]
```diff
@@ -586,6 +584,72 @@ public static function general_page() {
...
+		 *
+		 * @since x.x.x
+		 *
+		 * @return string Starter Templates Plugin name.
+		 */
+		public static function get_starter_templates_title() {
+
+			$astra_sites_name = __( '150+ Starter Templates', 'astra' );
+
+			if ( is_callable( 'Astra_Ext_White_Label_Markup::get_whitelabel_string' ) ) {
```
Comment (rohitp-bsf) [inline inc/core/class-astra-admin-settings.php:643]: Sure to use `is_callable`, I think it works with access specifiers only. 
ex: https://www.php.net/manual/en/function.is-callable.php

Good to use `method_exists`, just cross-verify it once!

**Review by rohitp-bsf — CHANGES_REQUESTED** [review]

File: inc/core/class-astra-admin-settings.php (hunk 2 above)
Comment (imnavanath) [reply inc/core/class-astra-admin-settings.php]: Updated with `method_exists` & verified with after whitelable, with free theme like these cases.

**Review by Nikschavan — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**